spec/rubocop/cop/lint/duplicate_methods_spec.rb

```diff
@@ -0,0 +1,149 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Lint::DuplicateMethods do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for duplicate method in class' do
```

Comment:
I'd like to see an example where two different method names are duplicated, e.g., two `some_method`s and two `some_other_method`s. We should get two offenses. And compare `cop.messages` in at least one place, not only `cop.offenses.size`.